src/pages/library/DataBrowser.js

```diff
@@ -42,9 +42,10 @@ const getUnique = (prop, data) => _.flow(
 // Description of the structure of the sidebar. Case is preserved when rendering but all matching is case-insensitive.
 const extractCatalogFilters = dataCatalog => {
   const accessArray = _.values(snapshotAccessTypes)
+  const releasePolicyArray = _.map('tag', snapshotReleasePolicies)
```

Comment:
When I change `tag` to label, I end up seeing more restrictions in the filter column.  I am not sure of the purpose of the tag property prefix.

Reply:
You do, but it's inaccurate - if you click on the restruction (like "CC"), and then select the one dataset that matches, the json returned for that dataset actually shows the data use policy to be "NRES (no restriction)" as well.

The reason why I had to create the tag field is because of how the filters work - Each dataset just has an array of filter attributes that it matches on. When we select a filter, it checks the string to see if that value exists, but it doesn't check if it matches for that same field.

So back to the example, the "CC" (release policy filter) will match on the dataset "Single Cell transcriptomic landscape of early human diabetic nephropathy" because the disease is "Enteroco**cc**us faecalis infection". It's only doing a simple string match.

I would recommend eventually rewriting how these filters work, but since we're building on code that already existed in the library, that felt like a bigger update